pkg/utils/constants/constants.go

```diff
@@ -56,4 +56,10 @@ const (
 	Read    = "Read"
 	Write   = "Write"
 	Execute = "Execute"
+
+	//Signature parameter name
```

Comment:
these constants is only used for upload to snapshot to cloud, please move to contrib/backup/multi-cloud directory.

Reply:
but as erik said. we should use public lib